Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -190,13 +193,28 @@ select2 personId estimateId req@DSelectReq {..} = do
         providerUrl = estimate.providerUrl,
         variant = DVST.castServiceTierToVariant estimate.vehicleServiceTierType, -- TODO: fix later
         isAdvancedBookingEnabled = fromMaybe False isAdvancedBookingEnabled,
+        taggings = if isValueAddNP then getTag remainingEstimateBppIds else Nothing,
```

Comment:
lets abstract this `isValueAddNP` information also in instance, will discuss tomorrow.